scripts/GSAS-II/install_gsas_proxy.py

```diff
@@ -0,0 +1,101 @@
+import argparse
+import os
+import pip
+import subprocess
+import urllib2
+import zipfile
+
+
+GSAS_SVN_URL = "https://subversion.xray.aps.anl.gov/pyGSAS/install/GSASIIproxy.zip"
+GSAS_BOOTSTRAP_URL = "https://subversion.xray.aps.anl.gov/pyGSAS/install/bootstrap.py"
+GSAS_PROXY_FILE_NAME = "GSASIIProxy.zip"
+GSAS_HOME_DIR_NAME = "g2conda"
+
+
+def download_zip_file(target_location):
+    response = urllib2.urlopen(GSAS_SVN_URL)
+    zip_file = response.read()
+    response.close()
+
+    with open(target_location, "wb") as out:
+        out.write(zip_file)
+
+
+def unzip_file(zip_file_name, target_directory):
+    zip_file = zipfile.ZipFile(zip_file_name, "r")
+    zip_file.extractall(target_directory)
+    zip_file.close()
+
+
+def download_bootstrap(revision_number, target_location):
+    url = GSAS_BOOTSTRAP_URL
+    if revision_number is not None:
+        url += "?r={}".format(revision_number)
+
+    response = urllib2.urlopen(url)
+    bootstrap_file = response.read()
+    response.close()
+
+    with open(target_location, "w") as out_file:
+        out_file.write(bootstrap_file)
+
+
+def gsas_install_directory():
+    root_dir = os.path.abspath(os.sep)
```

Comment:
I just had a thought while looking over this again. Python has the concept of a user site directory where users can install packages and Python automatically knows how to pick them up without any additional configuration of `sys.path`. The module `site` defines the [`USER_SITE`](https://docs.python.org/2/library/site.html#site.USER_SITE)) variable to point at this location.

Perhaps that is a better default for the installation location? The system test would then not need to do any directory searching and `def skipTests` can just check if `import GSASIIscriptable` work without raising an `ImportError`?

Reply:
I prefer this as a default for the installation location, as this way if a user installs GSAS-II via the script then it's in a convenient place for them to use outside of the algorithm, instead of being squirelled away somewhere weird

Reply:
I see your point - that's fine. Could we swap it around though so that the python script default location is `site.USER_SITE` and then the install bat script puts it into the location that is better for the users? That should have the same effect?

I'd like the system tests to not have to search for the folder and just test for `import GSASIIscriptable` as it would be better if the system test didn't have hardcoded values like "look 3 levels way etc". It makes the tests harder to run on systems that are not setup like that.

Reply:
Yup, that sounds like a more sensible way of doing it!